podcasts/ProfileIntroViewController.swift

```diff
@@ -10,14 +11,24 @@ class ProfileIntroViewController: PCViewController, SyncSigninDelegate {
         }
     }
 
+    @IBOutlet var authenticationProviders: UIStackView!
     @IBOutlet var signInBtn: ThemeableRoundedButton! {
         didSet {
+            signInBtn.isHidden = FeatureFlag.signInWithApple
             signInBtn.setTitle(L10n.signIn, for: .normal)
             signInBtn.shouldFill = false
             signInBtn.titleLabel?.font = UIFont.systemFont(ofSize: 18, weight: UIFont.Weight.semibold)
         }
     }
 
+    @IBOutlet var passwordAuthOption: ThemeableUIButton! {
+        didSet {
+            passwordAuthOption.isHidden = !FeatureFlag.signInWithApple
+            passwordAuthOption.setTitle(L10n.accountLogin, for: .normal)
+            passwordAuthOption.titleLabel?.font = UIFont.systemFont(ofSize: 18, weight: UIFont.Weight.semibold)
```

Comment:
You can simplify this to:

```suggestion
            passwordAuthOption.titleLabel?.font = .systemFont(ofSize: 18, weight: .semibold)
```

Reply:
I'll apply that pattern to the other buttons on this screen to then push the change